HISTORY.rst

```diff
@@ -22,6 +22,10 @@ API Changes (Backward-Compatible)
   a subclass of ``int``, this is non-breaking.
 - Deprecated the other fields in ``h2.settings``. These will be removed in
   3.0.0.
+- Added a new parameter to the ``H2Configuration`` initializer which takes a
+  logger, and if none given uses ``DummyLogger``.  This allows us to log by
```

Comment:
Let’s put in the full path the class, cf the new Settings object a few lines above.